private_gpt/ui/ui.py

```diff
@@ -55,9 +55,11 @@ class Source(BaseModel):
     file: str
     page: str
     text: str
+    pdf_prefix: str = "C:/UsedFilesFolder/"
```

Comment:
you cannot assume that. You will need to store files in a folder in addition to current process